assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
```

Comment:
Rather than adding the new CSS class and definition `medium-width`, simply add `max="99999"` (or, in the case of a `text` input, `maxlength="5"`. The input size will adjust.